force-app/main/default/objects/Journey_Task__c/fields/Relative_Days_to_Complete__c.field-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Relative_Days_to_Complete__c</fullName>
+    <description>This is the predicted amount of days, relative to each Mainlydigital candidate that they require to successfully complete their onboarding tasks.</description>
```

Comment:
An example of making concise descriptions, this can be cut down to something along the lines of:

> The predicted amount of days for each candidate to complete a task.